react/README.md

```diff
@@ -528,6 +534,51 @@ This guide provides a few guidelines on writing sensible React. Many of these ru
 
 
 
+## Modules
+
+- [7.1](#7.1) <a name="7.1"></a> When importing from React, explicitly import the exact features you need (in addition to React, which will be a signal to the compiler to compile JSX).
+
+  > Why? Named imports explicitly indicate what you will be using at the top of the file, and reduce the duplication of accessing properties on `React`.
+
+  ```js
+  // bad
+  import React from 'react';
+
+  class MyComponent extends React.Component {
+    static propTypes = {object: React.PropTypes.object};
+  }
+
+  // good
+  import React, {Component, PropTypes} from 'react';
+
+  class MyComponent extends Component {
+    static propTypes = {object: PropTypes.object};
+  }
+  ```
+
+- [7.2](#7.2) <a name="7.2"></a> Try to define only one React component per file. Name that file the same as the component it exports. Any subcomponents can be exposed as statics on that component. Finally, use an `index.js` to make it easier for other components to import.
+
+  ```js
+  // in Card/Section.js
+  export default class Section extends Component {}
+
+  // in Card/Card.js
+  import Section from './Section';
+
+  export default class Card extends Component {
+    static Section = Section;
```

Comment:
I just don't really like it because it kind of goes around the ES6 module system and depends on adding properties to things that, although they're related, don't _need_ that property. It feels like kind of a step backwards